### PR TITLE
Add prow config with one job for faster testing of perfdash

### PR DESCRIPTION
**Pull request by marseel**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Make it easier and faster to test perfdash locally.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: perfdash/test/jobs.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+periodics:
+- cron: '1 17 * * *' # Run daily at 9:01PST (17:01 UTC)
+  name: ci-kubernetes-e2e-gce-scale-performance
+  tags:
+  - "perfDashPrefix: gce-5000Nodes"
+  - "perfDashBuildsCount: 270"
```
Comment (tosi3k) [inline perfdash/test/jobs.yaml:6]: I think we could accelerate running `make run` without the `perfDashBuildsCount` override, WDYT?

Comment (marseel) [reply perfdash/test/jobs.yaml]: Done.

**Pull request merged**